freqtrade/optimize/hyperopt.py

```diff
@@ -75,8 +75,8 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         self.trials_file = (self.config['user_data_dir'] /
                             'hyperopt_results' / 'hyperopt_results.pickle')
-        self.tickerdata_pickle = (self.config['user_data_dir'] /
-                                  'hyperopt_results' / 'hyperopt_tickerdata.pkl')
+        self.data_pickle_file = (self.config['user_data_dir'] /
+                                 'hyperopt_results' / 'hyperopt_data.pkl')
```

Comment:
i guess this renaming is ok as it's rebuilt each time.

but assuming a user already used hyperopt before - won't this leave an "orphaned" file with the old filename on the disk it's never going to cleanup again?

Reply:
returned back, I have some other possible plans for it for the far perspective, when it will be more convenient to change the name...